### ISSUE TITLE
Loading Google Fonts is not GDPR compliant

### ISSUE THREAD

**Issue by janmonschke** [usebruno/bruno#695]
According to a German court, loading Google Fonts from Google's servers is not GDPR compliant and can lead to fines ([source](https://www.cookieyes.com/documentation/features/google-fonts-and-gdpr/)).

Therefore, bruno should embedd the `Inter` Google font that is currently loaded from Googles servers.

https://github.com/usebruno/bruno/blob/f8f38802a97f7c22134e25c1b218c5f7ce96f6eb/packages/bruno-app/src/pages/_document.js#L34-L37

NextJS 13 provides a helper module that lets you embedd the font: https://nextjs.org/docs/app/building-your-application/optimizing/fonts#google-fonts. Currently bruno use Next 12.

**Comment by grubersjoe** [comment]
I have had good experience with https://www.npmjs.com/package/@fontsource/inter. Simply import the font and you're good to go.

**Comment by janmonschke** [comment]
@grubersjoe I agree that fontsource would solve this issue as well. Being in the next ecosystem, it might make sense to stick to the next modules though. (see #696)

**Comment by grubersjoe** [comment]
Agreed. I forgot that Bruno uses Next internally.

**Comment by helloanoop** [comment]
There is a v1.0.0 planned on Oct 29th 2023. Upgrading to a major version of Next with days away is a bit risky.

I welcome and support this [PR](https://github.com/usebruno/bruno/pull/696). Will review this post V1 launch.
Meanwhile, if there is a smaller PR that can introduce the inline google font without upgrading next, please raise a PR for the same.

**Comment by janmonschke** [comment]
@helloanoop Fixed it without the nexst update here: https://github.com/usebruno/bruno/pull/750

**Comment by grubersjoe** [comment]
Maybe using the @fontsource/inter package wouldn't be the worst idea. Then it work offline and Google is out if the equation.

**Comment by janmonschke** [comment]
Agreed, I updated the PR because, in the meantime, conflicts occured in the package-lock file. https://github.com/usebruno/bruno/pull/750

**Comment by helloanoop** [comment]
Reopening this issue. 
With Pr #750 , the Inter font loaded perfectly in development versions but was not working in production build.

**Comment by NiklasBr** [comment]
Maybe use the [system fonts](https://fullystacked.net/system-font-deep-dive/) if Inter is problematic?

**Comment by helloanoop** [comment]
We are now using `@fontsource/inter`
https://github.com/usebruno/bruno/blob/main/packages/bruno-app/src/pages/Main.js#L16-L24

Closing this issue.
